src/channels/base/conduit.ts

```diff
@@ -76,7 +76,7 @@ export abstract class ConduitInstance<TConfig, TContext extends ChannelContext<a
         try {
           await sender.send(context)
         } catch (err) {
-          this.logger.error('An error occurred when sending a message', err)
+          this.logger.error('An error occurred when sending a message:', (err as Error).message)
```

Comment:
I figured if this worked here that I'd also add this for the renderer errors

Reply:
The reason I did not do the same thing for the sender is because the renderer will never print sensitive information in its error stack